src/miden-tutorials/web-client/about.md

```diff
@@ -0,0 +1,12 @@
+# Web Client
+
+The Miden Web client can be used for a variety of things, including:
```

Comment:
Typescript library, which can be used to programmatically interact with the Miden rollup.